README.md

```diff
@@ -8,13 +8,14 @@ To use the extension, press `Cmd + Shift + Space` and search for the bookmark.
 
 Since this is unfinished, it is not yet on the Chrome app store.  However, if you're eager to use the tool, you can use it by manually loading it.  To use this extension:
 - Clone the source
-- Run `npm install && npm run build`
-- You'll have to add your own icon to `src/img/icon.png`.  There will be one added soon, but until then Google search `search icon` and grab one that's around 128x128.
-- Import the extension in Chrome's extension tab using `Load unpacked extension`, pointing at the `bookmark-spotlight-react` folder
-
-### Notes
-
-There's still a lot of work to be done -- if you have any suggestions, feel free to open an issue.  Also, feel free to contribute!  :)
+- You'll have to add your own icon to `src/img/icon.png`.  There will be one added soon, but until then google search `search icon` and grab one that's around 128x128.
```

Comment:
I'd recommend adding a white/transparent 128x128 image for now, as a placeholder.